pkg/barbicankeystonelistener/deployment.go

```diff
@@ -0,0 +1,161 @@
+package barbicankeystonelistener
+
+import (
+	"fmt"
+
+	"github.com/openstack-k8s-operators/lib-common/modules/common"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/env"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	//"k8s.io/apimachinery/pkg/util/intstr"
+
+	barbicanv1beta1 "github.com/openstack-k8s-operators/barbican-operator/api/v1beta1"
+	barbican "github.com/openstack-k8s-operators/barbican-operator/pkg/barbican"
+)
+
+const (
+	// ServiceCommand -
+	ServiceCommand = "/usr/local/bin/kolla_start"
+)
+
+// Deployment - returns a Barbican Keystone Listener Deployment
+func Deployment(
+	instance *barbicanv1beta1.BarbicanKeystoneListener,
+	configHash string,
+	labels map[string]string,
+	annotations map[string]string,
+) *appsv1.Deployment {
+	runAsUser := int64(0)
+	var config0644AccessMode int32 = 0644
+	envVars := map[string]env.Setter{}
+	envVars["KOLLA_CONFIG_STRATEGY"] = env.SetValue("COPY_ALWAYS")
+	envVars["CONFIG_HASH"] = env.SetValue(configHash)
+	/*
```

Comment:
Lets get rid of the commented out parts here.  we'll have to figure out probes for these kinds of objects (listener, worker) later, but they won't be these kinds of probes